src/libs/mrubyWriterConnector.ts

```diff
@@ -262,7 +263,11 @@ export class MrubyWriterConnector {
       await this.sendCommand("execute");
     }
 
-    return Success.value(null);
+    const crc = crc8Calculator(binary);
+    const crcRes = writeRes.value.split("+OK")[1];
```

Comment:
意図が分かりづらいので、`"".match(//)`を使って正規表現で取得するのがよさそうです。
```suggestion
    const crcRes = writeRes.value.match(/^\+OK (?<hash>[0-9a-zA-Z]+)$/)?.groups?.hash;
```
のような感じでしょうか？(テキトーに書いたのでちゃんと検証していません)